tensorflow/BUILD

```diff
@@ -395,39 +395,8 @@ py_library(
 )
 
 filegroup(
-    name = "all_opensource_files",
+    name = "all_contrib_files",
```

Comment:
Could you revert this change? we actually have some primitive checks on the all_opensource_files rule, and the flat structure simplifies things.

Reply:
Is there a more efficient way to require the filetree inside `contrib` ?